scripts/upgrade

```diff
@@ -169,6 +169,13 @@ if ynh_compare_current_package_version --comparison lt --version 4.2.0~ynh1; the
 	popd
 fi
 
+if ynh_compare_current_package_version --comparison lt --version 5.0.0~ynh1; then
```

Comment:
Would that work for any 5.x version ?

Reply:
I have no idea... just sticking to the pattern